llms/llama/llama.py

```diff
@@ -330,7 +331,24 @@ def sanitize_config(config, weights):
 
 def load_model(model_path):
     model_path = Path(model_path)
-    weights = mx.load(str(model_path / "weights.npz"))
+
+    unsharded_weights_path = Path(model_path / "weights.npz")
+    if unsharded_weights_path.is_file():
+        print("[INFO] Loading model from {}.".format(unsharded_weights_path))
+        weights = mx.load(str(model_path / "weights.npz"))
+    else:
+        sharded_weights_glob = str(model_path / "weights.*.npz")
+        weight_files = glob.glob(sharded_weights_glob)
+        print("[INFO] Loading model from {}.".format(sharded_weights_glob))
+
+        weight_files = sorted(weight_files, key=lambda tf: int(tf.split(".")[-2]))
```

Comment:
I don't think it's necessary to load the files in sorted order since the weights are just going into the dictionary, this line can be removed?